src/hooks/voucherHelpers.js

```diff
@@ -0,0 +1,22 @@
+import { ethers } from 'ethers';
+
+export async function getVoucherFromServer() {
+    const tokenId = 1;
+
+    const expiry = Math.round(new Date().getTime() / 1000) + 1000;//timestamp
+    // const expiry = 1657335995;//timestamp
+    const nonce = ethers.BigNumber.from(ethers.utils.randomBytes(4)).toNumber();
+    // const nonce = 552311376;
+    const balance = 0;
+
+    const response = await fetch("http://172.32.13.15:8081/claim", {
```

Comment:
this needs to be broken out into a const, placed in constants.js, we probably need something like voucher.webaverse.com -- then we can do local loopback to test (add entry to /etc/hosts)